### ISSUE TITLE
Evalutate/activate donation github feature

### ISSUE THREAD

**Issue by robertobandini** [freelensapp/freelens#25]
As a community we need money to take care of some things like signing, website ecc..

**Comment by elephantum** [comment]
Yes, I was looking for a way to donate and could not find any :)

**Comment by robertobandini** [comment]
Thank you very much @elephantum! 
This is actually my current main task, so I hope to do it asap, today or anyway during this week!
I will also document needed expenses and how they are covered.
Any suggestion or contribution is appreciated, thank you!

**Comment by bo0tzz** [comment]
I'm looking forward to this option. Thank you very much for your effort in keeping this app alive, I haven't found anything else that can match it!

**Comment by robertobandini** [comment]
Thanks @bo0tzz ! I'll update you here asap!
In the meantime your appreciation and contribution is what is worth the most!

**Comment by robertobandini** [comment]
I did a little more analysis and I think there is much more to verify regarding the donations area and the use of systems like Open Collective.

To begin with I activated GitHub Sponsors, we will enable it on the various repositories, I started with the main one.
You can try it if you want to offer us a coffee, so we can see if it works correctly.

I will also prepare a page at least regarding the expenses currently incurred, which in any case are estimated to be a maximum of 250 euros per year between signing and web hosting which I am providing directly.

For the future in the possible scaling of the project we will better evaluate the use of systems like Open Collective and similar if necessary.

Anyone with suggestions regarding the topic is always welcome.

Thanks regardless for your contribution and your support for the project!

**Comment by elephantum** [comment]
Nice! Let's see if it works!

**Comment by robertobandini** [comment]
It seems to be working for now!
I'll keep the issue open while I finish activating it and writing the page about it.

Thank you very much @bo0tzz and @elephantum, on behalf of me, but especially @dex4er and @mariomamo, thanks to which we are making great progress!

**Comment by robertobandini** [comment]
I created this page for now, later we will see whether or not to use systems like Open Collective or evolve in another way.

https://github.com/freelensapp/freelens/wiki/Expenses-and-Donations

The Readme has also been updated with the indication to this page in the Wiki.

Thanks again to everyone and if anyone has any suggestions let us know!
